docs/sdk-docs/private-chain/README.md

```diff
@@ -0,0 +1,144 @@
+
+ * `deployment_private_net.json`
+ * `deployment_services_private_net.json`
+
+Copy these files to your project's `src/deployment` and proceed with the SDK or dApp initialization.
+
+### Starting the SDK
+You can start using the SDK in your setup.
+
+To start the SDK on your private network setup, you need to import the private deployment files. You can find them in the `<rootDir>/src/deployment` directory.
+
+```typescript
+import privateDeployment from '@/deployment/deployment_private_net.json';
+import privateServicesDeployment from '@/deployment/deployment_services_private_net.json';
+import { ContractsInfo } from 'raiden-ts';
+```
+
+As soon as the imports exist, you need to construct the `contractsInfo` object and pass it to the Light Client SDK initialization logic. 
+
+An example of the SDK initialization function follows:
+
+```typescript
+async function initialization() {
+    const contractsInfo = ({
+      ...privateDeployment.contracts,
+      ...privateServicesDeployment.contracts
+    } as unknown) as ContractsInfo;
+
+    const raiden = await Raiden.create(
+        provider,
+        0,
+        window.localStorage,
+        contractsInfo
+    );
+}
+```
+
+Calling the `initialization()` function should enable the SDK to manage channels and send payments in the development environment, using the previously deployed token.
+
+### Changing the dApp to work on the private setup
+Out of the box the dApp supports the following public test networks `Görli`, `Rinkeby` and `Ropsten`.
+
+To run the dApp on any other network setup you need to deploy the smart contracts and change the dApp to use this setup.
+
+If you require help running the dApp, you can follow the [instructions].
+
+As soon as you have the dApp running you can locate the [`createRaiden`] function and change it accordingly. 
+
+First, you need to import the deployment information, as with the SDK setup.
+
+```typescript
+import privateDeployment from '@/deployment/deployment_private_net.json';
+import privateServicesDeployment from '@/deployment/deployment_services_private_net.json';
+import { ContractsInfo } from 'raiden-ts';
+```
+
+Then you need to construct the `contractsInfo` object. 
+
+The [`createRaiden`] function should look like the example below:
+
+```typescript
+  private static async createRaiden(
+    provider: any,
+    account: string | number = 0
+  ): Promise<Raiden> {
+    const contractsInfo = ({
+      ...privateDeployment.contracts,
+      ...privateServicesDeployment.contracts
+    } as unknown) as ContractsInfo;
+    try {
+      return await Raiden.create(
+        provider,
+        account,
+        window.localStorage,
+        contractsInfo,
+        { pfsSafetyMargin: 1.1 }
+      );
+    } catch (e) {
+      throw new RaidenInitializationFailed(e);
+    }
+  }
+```
+
+To finalize the setup, you also need to change MetaMask's RPC provider. Simply go to the network dropdown and select `Localhost 8545`. 
```

Comment:
A pointer that the user needs to open their MetaMask so that they don't go looking for this in the LC source code?

Reply:
Sure I will add it.